### ISSUE TITLE
Endpoints???

### ISSUE THREAD

**Issue by slaugh** [xtrinch/fcm-django#15]
Hello,

I love that you guys have created an FCM wrapper for django that supports DRF. However after setting this up I realize that there I don't see any endpoints that I can call to test this tool. Is there any reason for that? Do I need to look into another library to find these endpoints???

Please let me know :)

**Comment by xtrinch** [comment]
If you want a list of all available endpoints, add `router.register(r'devices', FCMDeviceAuthorizedViewSet)` and when you visit <api_root>/devices in your browser, you should see a graphical browsable API. 

Did I understand the question correctly & does that answer your question?

**Comment by slaugh** [comment]
So I see now that your get this nice rest_frameworkey browsable API when I go to <api_roo>/devices. Very cool! 

I did struggle with using this api. It seems that there is only one endpoint and it was not clear to me from the documentation on how to use it or how to create an FCMDevice object using this API. I ended up creating my own endpoint where I create my own FCMDevice object and create notifications using device.send(). I think this library is great, and I think it would be even better if the documentation was a bit more clear on how to call the API. This will now be my go to library for Django + Notifications :)

Best,

Scott
